frontend/src/screens/Login/accountSettings.tsx

```diff
@@ -25,6 +25,8 @@ export default function AccountSettings() {
         <ErrorText message={error} />
         <Text>{successMessage}</Text>
         <TouchableOpacity onPress={() => {
+            setError("");
+            setSuccesMessage("");
```

Comment:
Can we fix this typo for setSuccessMessage?

Reply:
Fixed!